### PR TITLE
Add workflow prettier js

### PR DESCRIPTION
**Pull request by ghost**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the documentation and the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**
Add a coding style checker for javascript.

**Test plan**

...

**Closing issues**

...

### CODE REVIEW

File: shlr/rizin-shell-parser/grammar.js [hunk 1]
```diff
@@ -1,658 +1,728 @@
...
+          alias($._search_command, $.arged_command)
+        )
+      ),
+    iter_interpret_command: ($) =>
+      prec.right(1, seq($._simple_command, "@@c:", $._simple_command)),
+    iter_interpret_offsetssizes_command: ($) =>
+      prec.right(1, seq($._simple_command, "@@@c:", $._simple_command)),
+    iter_comment_command: ($) =>
+      prec.right(1, seq($._simple_command, "@@C", optional(seq(":", $.arg)))),
+    iter_dbta_command: ($) =>
```
Comment (ret2libc) [inline shlr/rizin-shell-parser/grammar.js]: what is the max width? I think it should be increased a bit... Having these lines differently is a bit confusing.

Comment (ghost) [reply shlr/rizin-shell-parser/grammar.js]: 80
I can change them to 120 if you want.

Comment (ret2libc) [reply shlr/rizin-shell-parser/grammar.js]: yes please. at least 100.

Comment (ghost) [reply shlr/rizin-shell-parser/grammar.js]: I move it to 120 to simplify lambda declaration.

**Comment by ghost** [comment]
PS: this job should be tag as required.

**Review by ret2libc — APPROVED** [review]

**Pull request merged**